### PR TITLE
Added Acquisition Overview

### PR DESCRIPTION
**Pull request by jdamis4XD**
Acquisition overview content has been added

## Summary
- Resolves [# ]

_Include a summary of proposed changes._

[:sunglasses: PREVIEW LINK](https://federalist-proxy.app.cloud.gov/preview/18f/cic-site/Added-AcqOverview/)

## Impacted areas of the application
_List general components of the application that this PR will affect:_

-  Acquisition

## Screenshots
_Include a screenshot of the new/updated features in context (“in the wild”). If it is an interface or front end change, include both a Before and After screenshot._

### CODE REVIEW

File: pages/acquisition.md [hunk 1]
```diff
@@ -10,11 +10,15 @@ link_cards:
...
+* **Research different vehicles**. There are multiple ways to acquire cloud solutions, and each has its own strengths. [Within GSA alone](https://www.gsa.gov/technology/technology-programs) you can choose from IT Schedule 70, government-wide acquisition contracts (GWACs), and software blanket purchase agreements (BPAs). 
+* **Explore Acquisition Gateway**. [Acquisition Gateway](https://hallways.cap.gsa.gov/login-information) offers resources and contacts for both contract officers and buyers. Within the tool you can see current and past successful contracts, as well as points of contact to learn more.
+* **Share best practices**. The Cloud Center of Excellence (CCoE) has created the Cloud Adoption Survival Tips, Lessons Learned, and Experiences (CASTLE) guide, an extensive resource for acquisition professionals. Ask your contracting officers if they’ve seen it, and if not feel free to share.
```
Comment (lauraGgit) [inline pages/acquisition.md]: is there a link to this reference?

Comment (jdamis4XD) [reply pages/acquisition.md]: @abbeykos updated the content since I posted so there is no longer a reference to CASTLE. I need to update the branch

**Comment by jdamis4XD** [comment]
@lauraGgit updated with final content

File: pages/acquisition.md [hunk 2]
```diff
@@ -10,11 +10,16 @@ link_cards:
...
+* **Explore Acquisition Gateway**. [Acquisition Gateway](https://hallways.cap.gsa.gov/login-information) offers resources and contacts for both contract officers and buyers. Within the tool you can see current and past successful contracts, as well as points of contact to learn more.
+* **Share best practices**. Ask colleagues at other agencies about what’s worked for them. There may be templates or guidance they can share. Also, our [Process](/acquisition/procurement-process/) page can help guide acquisition officers who haven’t worked with cloud contracts before.
+* **Don’t blame acquisition for other challenges**. Because cloud migration is complex, it can be tempting to assign all the blame to contracting. But many of the individual steps, like taking inventory or understanding pricing differences between vendors, are less about contracts than they are about the process itself.
```
Comment (lauraGgit) [inline pages/acquisition.md]: ha, love this!

**Pull request merged**